src/.vuepress/config.js

```diff
@@ -51,7 +51,7 @@ const sidebar = {
       ]
     },
     {
-      title: '可复用性&组合',
+      title: '可复用性&组合式',
```

Comment:
```suggestion
      title: '可复用&组合',
```

这里的原文不是完整的 Composition API 我觉得可以灵活处理一下，会觉得更通顺一些